db/schema.rb

```diff
@@ -222,31 +230,31 @@
   create_table "versions", id: :serial, force: :cascade do |t|
     t.text "authors"
     t.text "description"
-    t.string "number"
+    t.string "number", limit: 255
     t.integer "rubygem_id"
-    t.datetime "built_at"
-    t.datetime "updated_at"
+    t.datetime "built_at", precision: nil
+    t.datetime "updated_at", precision: nil
     t.text "summary"
-    t.string "platform"
-    t.datetime "created_at"
+    t.string "platform", limit: 255
+    t.datetime "created_at", precision: nil
     t.boolean "indexed", default: true
     t.boolean "prerelease"
     t.integer "position"
     t.boolean "latest"
-    t.string "full_name"
+    t.string "full_name", limit: 255
+    t.string "licenses", limit: 255
     t.integer "size"
-    t.string "licenses"
     t.text "requirements"
-    t.string "required_ruby_version"
-    t.string "sha256"
+    t.string "required_ruby_version", limit: 255
+    t.string "sha256", limit: 255
```

Comment:
I don't think a limit adds a lot of value here, given the way PostgreSQL already intelligently handles strings.